.github/workflows/deploy.yml

```diff
@@ -6,6 +6,14 @@ env:
   DOCKER_IMAGE: docker.pkg.github.com/${{ github.repository }}/divergent-console
   DOCKER_TAG: latest
 jobs:
+  test-app:
```

Comment:
configured this as a separate job in the deploy pipeline. ideally would move this to the pull-request pipeline to run against a local server